public/pages/AnomalyCharts/components/FeatureChart/FeatureChart.tsx

```diff
@@ -278,9 +280,29 @@ export const FeatureChart = (props: FeatureChartProps) => {
                   yAccessors={[CHART_FIELDS.DATA]}
                   data={featureTimeSeries}
                 />
-              );
+              )
+              if (featureTimeSeries.map(
+                (item: FeatureAggregationData) => {
+                  if(item.hasOwnProperty('expectedValue')) {
+                    componentList.push(
+                      <LineSeries
+                        id={"ExpectedValue"}
+                        name={"Expected Value"}
+                        color={"#0475a2"}
```

Comment:
What is this color? Can you add a comment in the code about it somewhere? If supposed to be invisible, may need to find a way to always keep it consistent with the chart background, including if Dashboards is in dark mode or not

Reply:
It's a bluish color and shouldn't be invisible. I need to attend the UX office hour next Tuesday to get this color finalized.

Reply:
Ok - is there a way for you to work with them sooner? Concern is that 8/9 is scheduled as the final sanity testing day - see timeline https://github.com/opensearch-project/opensearch-build/issues/2271

Reply:
sure, will reach out to them offline to draw an conclusion on this